sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/aggregate/interfaces.scala

```diff
@@ -126,7 +126,13 @@ private[sql] case class AggregateExpression(
     AttributeSet(childReferences)
   }
 
-  override def toString: String = s"($aggregateFunction,mode=$mode,isDistinct=$isDistinct)"
+  override def toString: String = {
+    if (isDistinct) {
+      s"$aggregateFunction(distinct, $mode)"
```

Comment:
this is different from what we discussed before, we need something like `partitial_sum(distinct a)`, instead of `sum(a)(distinct, partitial)`